app-metrics/prom2json/prom2json-0.1.0_p20170523.ebuild

```diff
@@ -1,8 +1,8 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
-inherit user golang-build golang-vcs-snapshot
```

Comment:
New line after EAPI for m-n and sort inherit if it doesn't break phase export order (if stuff still gets called as needed). You can skip this if not m-n as it's style though